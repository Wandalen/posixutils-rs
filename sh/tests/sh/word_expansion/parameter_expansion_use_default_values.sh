VAR=value NULL=''

echo ${VAR:-default}
echo ${NULL:-default}
echo ${unset:-default}
echo $VAR $NULL

echo ${VAR-default}
echo ${NULL-default}
echo ${unset-default}
echo $VAR $NULL