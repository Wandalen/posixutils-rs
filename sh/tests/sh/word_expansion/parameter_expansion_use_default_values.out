value
default
default
value
value

default
value
